monai/utils/nvtx.py

```diff
@@ -109,7 +109,16 @@ def range_wrapper(*args, **kwargs):
             return output
 
         # Replace the method with the wrapped version
-        setattr(owner, method, range_wrapper)
+        if method.startswith("_"):
```

Comment:
The special method name has double underscores, could you please confirm the if condition here?

Reply:
That's right. Previously I wanted to capture `_call_impl` but I changed the logic. Thanks for capturing this. :)